### PR TITLE
fix(#54): add named route, update view and controllers

### LINKED ISSUE #54: Action App\Http\Controllers\romanzipp\QueueMonitor\Controllers\PurgeMonitorsController not defined
Error: 

```
InvalidArgumentException
Action App\Http\Controllers\romanzipp\QueueMonitor\Controllers\PurgeMonitorsController not defined. (View: /home/vagrant/ngcloud/vendor/romanzipp/laravel-queue-monitor/views/jobs.blade.php)

```
when enabling purge in config file. Same happens with allow deletion. the `action()` function is unable to fiind the action for the controller class because the strings needs to start with `\`

### PR DESCRIPTION
**Pull request by huzaifaarain**
- Updated all routes using Name Routes
- Removed `action()` and added `route()` function in views
- Removed `action()` and added `route()` function in controllers 

Fixes #54

### CODE REVIEW

**Review by sabinico — APPROVED** [review]
Reviewed and working fine in Laravel 5.8

File: src/Controllers/DeleteMonitorController.php [hunk 1]
```diff
@@ -11,6 +11,6 @@ public function __invoke(Request $request, Monitor $monitor)
     {
         $monitor->delete();
 
-        return redirect()->action(ShowQueueMonitorController::class);
+        return redirect()->route("romanzipp.showQueueMonitor");
```
Comment (romanzipp) [inline src/Controllers/DeleteMonitorController.php:14]:
```suggestion
        return redirect()->route('queue-monitor::index');
```

File: src/Controllers/PurgeMonitorsController.php [hunk 2]
```diff
@@ -16,6 +16,6 @@ public function __invoke(Request $request)
             $monitor->delete();
         }, 200);
 
-        return redirect()->action(ShowQueueMonitorController::class);
+        return redirect()->route('romanzipp.showQueueMonitor');
```
Comment (romanzipp) [inline src/Controllers/PurgeMonitorsController.php:19]:
```suggestion
        return redirect()->route('queue-monitor::index');
```

File: src/Routes/QueueMonitorRoutes.php [hunk 3]
```diff
@@ -15,14 +15,14 @@ public function queueMonitor(): Closure
     {
         return function (array $options = []) {
             /** @var \Illuminate\Routing\Router $this */
-            $this->get('', '\romanzipp\QueueMonitor\Controllers\ShowQueueMonitorController');
+            $this->get('', '\romanzipp\QueueMonitor\Controllers\ShowQueueMonitorController')->name("romanzipp.showQueueMonitor");
```
Comment (romanzipp) [inline src/Routes/QueueMonitorRoutes.php:18]:
```suggestion
            $this->get('', '\romanzipp\QueueMonitor\Controllers\ShowQueueMonitorController')->name('queue-monitor::index');
```

File: src/Routes/QueueMonitorRoutes.php [hunk 4]
```diff
@@ -15,14 +15,14 @@ public function queueMonitor(): Closure
...
 
             if (config('queue-monitor.ui.allow_deletion')) {
-                $this->delete('monitors/{monitor}', '\romanzipp\QueueMonitor\Controllers\DeleteMonitorController');
+                $this->delete('monitors/{monitor}', '\romanzipp\QueueMonitor\Controllers\DeleteMonitorController')->name("romanzipp.deleteMonitorEntry");;
```
Comment (romanzipp) [inline src/Routes/QueueMonitorRoutes.php:21]:
```suggestion
                $this->delete('monitors/{monitor}', '\romanzipp\QueueMonitor\Controllers\DeleteMonitorController')->name('queue-monitor::destroy');
```

File: src/Routes/QueueMonitorRoutes.php [hunk 5]
```diff
@@ -15,14 +15,14 @@ public function queueMonitor(): Closure
...
             }
 
             if (config('queue-monitor.ui.allow_purge')) {
-                $this->delete('purge', '\romanzipp\QueueMonitor\Controllers\PurgeMonitorsController');
+                $this->delete('purge', '\romanzipp\QueueMonitor\Controllers\PurgeMonitorsController')->name("romanzipp.purgeMonitorEntries");;
```
Comment (romanzipp) [inline src/Routes/QueueMonitorRoutes.php:25]:
```suggestion
                $this->delete('purge', '\romanzipp\QueueMonitor\Controllers\PurgeMonitorsController')->name('queue-monitor::purge');
```

File: views/jobs.blade.php [hunk 6]
```diff
@@ -188,7 +188,7 @@ class="w-64 text-xs p-1 border rounded"
 
                             <td class="p-4 text-gray-800 text-sm leading-5 border-b border-gray-200">
 
-                                <form action="{{ action(romanzipp\QueueMonitor\Controllers\DeleteMonitorController::class, $job) }}" method="post">
+                                <form action="{{ route('romanzipp.deleteMonitorEntry',[$job]) }}" method="post">
```
Comment (romanzipp) [inline views/jobs.blade.php:191]:
```suggestion
                                <form action="{{ route('queue-monitor::destroy', [$job]) }}" method="post">
```

File: views/jobs.blade.php [hunk 7]
```diff
@@ -278,7 +278,7 @@ class="w-64 text-xs p-1 border rounded"
 
         <div class="mt-12">
 
-            <form action="{{ action(romanzipp\QueueMonitor\Controllers\PurgeMonitorsController::class) }}" method="post">
+            <form action="{{ route('romanzipp.purgeMonitorEntries') }}" method="post">
```
Comment (romanzipp) [inline views/jobs.blade.php:281]:
```suggestion
            <form action="{{ route('queue-monitor::purge') }}" method="post">
```

**Review by romanzipp — CHANGES_REQUESTED** [review]

**Comment by romanzipp** [comment]
I've made some adjustments to the route prefixes by adding `::` as prefix separator to indicate that these routes are package provided and changed the prefix to `queue-monitor`

**Comment by romanzipp** [comment]
Thanks!

**Pull request merged**